### PR TITLE
Added documentation for workspace rules

### PR DESCRIPTION
**Pull request by levnikmyskin**
Hi,
this pull request adds documentation for the workspace rules defined in [Hyprland PR #1986](https://github.com/hyprwm/Hyprland/pull/1986). Hope everything looks ok.

Cheers

### CODE REVIEW

**Comment by vaxerski** [comment]
a ref from the explanation of the `workspace` keyword in the monitor section would be nice.

**Comment by vaxerski** [comment]
also, your reflinks are completely wrong, note to self before merging

**Comment by levnikmyskin** [comment]
Done. Links should also be fixed now :D

File: pages/Configuring/Workspace-Rules.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+### Examples
+```ini
+workspace=name:myworkspace,gapsin:0,gapsout:0
+workspace=DP-1,3,rounding:false,bordersize:0
+```
+
+## Rules
+| Rule | Description | type|
+| ---- | ----------- | ----|
+| gapsin:[x] | Set the gaps between windows (equivalent to [General->gaps_in](../Variables#general)) | int
```
Comment (vaxerski) [inline pages/Configuring/Workspace-Rules.md:27]: missing `|` at the end for all

Comment (levnikmyskin) [reply pages/Configuring/Workspace-Rules.md]: ups, fixed

**Review by vaxerski — APPROVED** [review]
lgtm

**Pull request merged**